dist/Dockerfile

```diff
@@ -1,18 +1,18 @@
 # Dockerfile
 
-FROM  java:8
+FROM anapsix/alpine-java:8_server-jre
 
-MAINTAINER  Jordan Halterman <jordan.halterman@gmail.com>
-
-RUN apt-get update && apt-get install -y iptables git stress
-
-RUN mkdir /atomix
+MAINTAINER  Jordan Halterman <jordan@opennetworking.org>
 
 ARG VERSION
-COPY target/atomix-agent-${VERSION}.tar.gz /atomix/atomix-agent-${VERSION}.tar.gz
-RUN tar -xvf /atomix/atomix-agent-${VERSION}.tar.gz -C /atomix && rm /atomix/atomix-agent-${VERSION}.tar.gz
+RUN mkdir -p /root/atomix
+COPY target/atomix-agent-${VERSION}.tar.gz /root/atomix/atomix-agent-${VERSION}.tar.gz
+RUN tar -xvf /root/atomix/atomix-agent-${VERSION}.tar.gz -C /root/atomix
+RUN rm /root/atomix/atomix-agent-${VERSION}.tar.gz
```

Comment:
better to chain both of these run commands with && to reduce the amount of layers.

Reply:
Ahh yes I knew there was some reason to do that